portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,13 @@ function addRandomFact() {
   const factContainer = document.getElementById('fact-container');
   factContainer.innerText = fact;
 }
+
+function fetchMessage() {
+    fetch("/data").then(request => request.json()).then((textResponse) => {
+        for (var i = 0; i < 3; i++) {
```

Comment:
Shall we display all the comments instead of hardcoding this to 3 comments?

Reply:
I have modified the condition of the for loop.